### ISSUE TITLE
Collapse/expand control for right sidebar (a la multireddit navbar)

### ISSUE THREAD

**Issue by andytuba** [honestbleeps/Reddit-Enhancement-Suite#3502]
https://www.reddit.com/r/Enhancement/comments/54hfeo/feature_request_grippy_for_right_side_bar/

Maybe autocollapse on small window sizes

**Comment by andytuba** [comment]
Stretch goal: remember to stay collapsed, per subreddit

**Comment by erikdesjardins** [comment]
If a control is added to the edge of the sidebar, care must be taken to avoid breaking subreddit styles. I could easily see it affecting absolutely-positioned sidebar elements (even when uncollapsed).

Because of that, and it being a de facto adblocker, and due to support requests when hiding the sidebar (re)moves other custom content, I'm not really in favour of implementing this.

Also note: if you know what you're doing [read: you accept that it will break stuff], this can be done manually with a custom toggle + a CSS snippet for `.side { display: none }`.

**Comment by honestbleeps** [comment]
with a common moderator complaint being that people don't read the rules especially on mobile (due to lack of sidebar) I also have a hard time supporting the ability to hide it.

**Comment by sparksis** [comment]
I've downloaded an extension (User CSS) and added the following to reddit.

```
@media (max-width: 960px) {
  .side {
    display: none;
  }
}

```

I think having a toggle in the options to inject this CSS would be amazing.  The media tag hides the side bar **when the screen size is less than or equal to half my screen width** (1080p).

I think this nicely addresses @honestbleeps issue of hiding the sidebar altogether while adding a quality of life improvement.  

PS. I have subreddit styles globally disabled so @erikdesjardins comment is still valid.

**Comment by williamparry** [comment]
Waiting on @andytuba to review my [pull request](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/4511).
